src/services/contracts/baseDAO/hooks/useProposeConfigChange.ts

```diff
@@ -32,7 +32,7 @@ export const useProposeConfigChange = () => {
         }
 
         const data = await dao.proposeConfigChange(args, tezosToolkit)
-        mixpanel.track("Proposal Created", {
+        AnalyticsService.track("Proposal Created", {
```

Comment:
Can we create Enums for the keys, so that there's no mistake in future with spellings and casings. 

If we are thinking of putting the same data everytime then i think it should be a function with defined params so that there's no inconsistency.